src/lambda/commands/createNewSamApp.ts

```diff
@@ -81,15 +84,14 @@ export async function createNewSamApp(
 }
 
 async function getMainUri(config: Pick<SamCliInitArgs, 'location' | 'name'>): Promise<vscode.Uri | undefined> {
-    try {
-        return await getMainSourceFileUri({
-            root: vscode.Uri.file(path.join(config.location.fsPath, config.name))
-        })
-    } catch (err) {
+    const samTemplatePath = path.resolve(config.location.fsPath, config.name, 'template.yaml')
```

Comment:
Why is `path.resolve` used instead of `path.join`?

Reply:
In general `path.resolve` always returns an absolute path while `path.join` could be either absolute or relative depending upon the arguments. Generally in programs, I prefer to pass around, log and print absolute paths as will work and be understood both internally and externally regardless of current working directory, hence I tend to favor `path.resolve` in most cases.